VERSION.yaml

```
@@ -1 +1 @@
-version: wip
+version: v0.3.0-alpha.1
```

I think "v" is redundant here

```suggestion
version: 0.3.0-alpha.1
```

---

Sure. My bad.